index.html

```diff
@@ -26,13 +29,13 @@ <h1>I'm Joseph Glad to meet you!</h1>
         </p>
             <p class="contain">LET’S CONNECT</p>
         
-        <div class="social"> 
-            <img src="Images/twitter.svg" alt="">
-            <img src="Images/Linkedin icon.svg" alt="">
-            <img src="Images/m vector.svg" alt="">
-            <img src="Images/github.svg" alt="">
-            <img src="Images/Vector.svg" alt="">
-        </div>
+        <ul class="social"> 
+            <li><img src="Images/twitter.svg" alt=""></li>
+            <li><img src="Images/Linkedin icon.svg" alt=""></li>
+            <li><img src="Images/m vector.svg" alt=""></li>
+            <li><img src="Images/github.svg" alt=""></li>
+            <li><img src="Images/Vector.svg" alt=""></li>
+        </ul>
```

Comment:
- [ ] [REQUIRED] Good job so far! So nice you implement `ul` and `li` but you missed implement **link(`a` tag)** because the list of icons uses a `ul`✔️ tag to group them, and `li`✔️  per icon; each icon has a `link`(`a` tag)❌. Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly implement my suggestion please 🙏🏼

Reply:
a tag has been added as requested.